packages/ui/form/radio-area/Radio.tsx

```diff
@@ -20,7 +20,7 @@ export const Radio = (props: RadioGroupPrimitive.RadioGroupItemProps & { childre
 export const Indicator = ({ disabled }: { disabled?: boolean }) => (
   <RadioGroupPrimitive.Indicator
     className={classNames(
-      "after:bg-default dark:after:bg-inverted relative flex h-full w-full items-center justify-center rounded-full bg-black after:h-[6px] after:w-[6px] after:rounded-full after:content-['']",
+      "after:bg-default relative flex h-full w-full items-center justify-center rounded-full bg-black after:h-[6px] after:w-[6px] after:rounded-full after:content-['']",
```

Comment:
I think we may need to take into consideration the active state for this. 

A user (even tho warned not to) can set a really bad accessable colour that doesnt pass the contrast checks. 

we could use something like dark:after:bg-brand-accent (not sure if its in our tailwind config but its at --cal-brand-accent) 

Sine we use brand colour for the selected BG it would make sense to have the right contrast colour there? Does that make sense

Reply:
Yes it does. Thanks @sean-brydon